src/server/search/doc_accessors.cc

```diff
@@ -26,13 +29,21 @@ string_view SdsToSafeSv(sds str) {
   return str != nullptr ? string_view{str, sdslen(str)} : ""sv;
 }
 
+string FtVectorToString(const search::FtVector& vec) {
+  return absl::StrCat("[", absl::StrJoin(vec, ","), "]");
+}
+
 }  // namespace
 
-string_view ListPackAccessor::Get(string_view active_field) const {
+string_view ListPackAccessor::GetString(string_view active_field) const {
   return container_utils::LpFind(lp_, active_field, intbuf_[0].data()).value_or(""sv);
 }
 
-SearchDocData ListPackAccessor::Serialize() const {
+search::FtVector ListPackAccessor::GetVector(string_view active_field) const {
+  return BytesToFtVector(GetString(active_field));
```

Comment:
Does `GetString` really return a buffer of binary packed floats? It's a pretty confusing name if it does...

Also, shame that we can't guarantee the alignment here :/

Reply:
The floats basically stored inside a hash field in raw byte format

Here is an example from RediSearch docs:
```
# Store a blob of a random vector of type float32 under a field named 'vector' in Redis hash.
np_vector = np.random.rand(dim).astype(np.float32)
redis_conn.hset('key', mapping = {vector_field: np_vector.tobytes()})
```

I need to parse it. GetString really returns a string - nothing wrong with its name. BytesToFtVector turns the string bytes to the vector